proposals/0007-access-policy.md

```diff
@@ -0,0 +1,159 @@
+```
+
+## Alternatives Considered
+
+[EIP](https://github.com/ethereum/EIPs) run under two separate levels,
+contributors and editors. Editors are a select group of people that review each
+EIP and have write access. Instead of starting with a select group of editors
+who have exclusive write access, we could begin with an easier way to achieve
+write access and gradually tighten permissions as needed.
+
+[RFCs](https://www.rust-lang.org/governance) have sub-teams that review and
+triage new proposed changes. This is a possibility, but would require more
+granular labeling of different SIMDs.
+
+[PEP](https://peps.python.org/pep-0013/) has an election of a council that
+reviews and triages new proposed changes. The PEP approach may result in an
+increase in governance overhead.
+
+## Detailed Design
+
+### Level 1 - Triage Access
+
+Requirements: One voucher from any user with level 2 or above access.
+
+Contributors with Triage access will have the associated [triage Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Improvement
+Documents](https://github.com/solana-foundation/solana-improvement-documents)
+repository. They will also have the Contributor role in the Solana Tech
+Discord. This is the lowest level of access.
+
+### Level 2 - Write Access
+
+Requirements: One voucher from anyone with level 3 access or two vouchers from
+anyone from level 2. Applicants must have level 1 access to apply for level 2.
+
+Contributors with Write access will have the associated [write Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Improvement
+Documents](https://github.com/solana-foundation/solana-improvement-documents)
+repository. They will also have the Core Contributor role in the Solana Tech
+Discord.
+
+Most notably these permissions allow the contributor to approve and merge pull
+requests.
+
+### Level 3 - Maintain Access
+
+Requirements: One voucher from anyone with level 3 access.
+
+Contributors will have the associated [maintain Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Improvement
+Documents](https://github.com/solana-foundation/solana-improvement-documents)
+repository. They will also have the Contributor role in the Solana Tech
+Discord.
+
+Users with Maintain access are responsible for managing the SIMD repository and
+granting user access based on the above requirements.
+
+### Vouching Process 
+
+To receive a level of access as defined in SIMD-0007, follow these steps:
+
+1. Open an issue on the [Solana Improvement
```

Comment:
In the interest of potentially assigning a Discord role in the future, the requester could also provide their discord id in the issue